### PR TITLE
Fixed iOS DismissProgressDialog Issue

### LINKED ISSUE #3218: iOS closes multiple open dialogs when one is closed
**Describe the bug**

[From the forum](https://community.appinventor.mit.edu/t/issue-with-the-dismissprogressdialog/121280): [Ferran_SIMON](https://community.appinventor.mit.edu/u/Ferran_SIMON) has posted an example app that opens a progress dialog and a choose dialog at the same time. They appear on top of each other in both Android and iOS, with whichever dialog was created last on top.

In Android, if you dismiss one dialog, you see the second one underneath. In iOS, if you dismiss one dialog, they both go away.

**Affects**

- [ ] Designer
- [ ] Blocks editor
- [ ] Projects Explorer
- [X] Companion (iOS)
- [X] Compiled apps
- [ ] Buildserver
- [ ] Debugging
- [ ] Other... (please describe)

**Expected behavior**

iOS and Android should behave as Android does now.

Also discussed in this issue: https://github.com/mit-cml/appinventor-sources/issues/3213

### PR DESCRIPTION
**Pull request by ghu999**
General items:

- [ ] I have updated the relevant documentation files under docs/
- [ ] My code follows the:
    - [ ] [Google Java style guide](https://google.github.io/styleguide/javaguide.html) (for .java files)
    - [ ] [Google JavaScript style guide](https://google.github.io/styleguide/jsguide.html) (for .js files)
- [ ] `ant tests` passes on my machine

If your code changes how something works on the device (i.e., it affects the companion):

- [ ] I branched from `ucr`
- [ ] My pull request has `ucr` as the base

Further, if you've changed the blocks language or another user-facing designer/blocks API (added a SimpleProperty, etc.):

- [ ] I have updated the corresponding version number in YaVersion.java
- [ ] I have updated the corresponding upgrader in YoungAndroidFormUpgrader.java (components only)
- [ ] I have updated the corresponding entries in versioning.js

For all other changes:

- [ ] I branched from `master`
- [ ] My pull request has `master` as the base

What does this PR accomplish?

Adds a CustomAlertView array to allow for handling of multiple dialogs. Before, the Dismiss progress dialog would not even appear on top of the choose dialog. Now, both work as intended based on the given app code; the dismiss dialog dismisses after 3 seconds and the choose dialog correctly cancels when clicked.

Fixes #3218.

Resolves # .

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by SusanRatiLane** [comment]
@AppInventorWorkerBee OK to test.

File: appinventor/appengine/src/com/google/appinventor/client/style/neo/neo.css [hunk 1]
```diff
@@ -606,10 +606,12 @@ they get cut off in Windows.
 
   .ode-Designer-LeftColumn {
     min-width: 265px;
+    flex-shrink: 0;
```
Comment (SusanRatiLane) [inline appinventor/appengine/src/com/google/appinventor/client/style/neo/neo.css:609]: You have correctly pointed this PR to ucr, but you branched from master. Therefore the diff has changes from master that are not currently in ucr. You need to also branch from ucr.

**Review by SusanRatiLane — CHANGES_REQUESTED** [review]

File: appinventor/components-ios/src/Notifier.swift [hunk 2]
```diff
@@ -262,13 +262,13 @@ open class Notifier: NonvisibleComponent {
...
-      }
-    }
-  }
+//  fileprivate var _activeAlert: CustomAlertView? = nil {
+//    didSet {
+//      if _activeAlert == nil {
+//        _alertType = nil
+//      }
+//    }
+//  }
```
[the remark this answers is not in the document]
Comment (ghu999) [reply appinventor/components-ios/src/Notifier.swift:271]: yes i did.

[the remark this answers is not in the document]
Comment (SusanRatiLane) [reply appinventor/components-ios/src/Notifier.swift:271]: > yes i did.

OK, if the code is not being used, it should be removed.

**Review by ewpatton — APPROVED** [review]
LGTM

**Pull request merged**